partitura/utils/generic.py

```diff
@@ -171,6 +171,35 @@ def iter_subclasses(cls, _seen=None):
                 yield sub
 
 
+def first_order_derivative(func, x0, dx=0.5):
```

Comment:
the default dx in the scipy function is 1. Is this different value on purpose?

Reply:
This is just the default value that was used in tempo by derivate which was the use of this function in first place. I am just propagating the default also here.